src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -793,6 +794,36 @@ namespace winrt::TerminalApp::implementation
         {
             _noMatchesText().Visibility(Visibility::Collapsed);
         }
+
+        if (_currentMode == CommandPaletteMode::CommandlineMode)
+        {
+            ParsedCommandLineText(L"");
+
+            const auto commandLine = _getTrimmedInput();
+            if (!commandLine.empty())
+            {
+                ExecuteCommandlineArgs args{ commandLine };
+                _appArgs.FullResetState();
+                if (_appArgs.ParseArgs(args) == 0)
+                {
+                    const auto& commands = _appArgs.GetStartupActions();
+                    if (commands.size() > 0)
+                    {
+                        std::wstring commandDescription{ RS_(L"CommandPalette_ParsedCommandLine") };
+                        for (const auto& command : commands)
+                        {
+                            commandDescription += L"\n\t" + command.Args().GenerateName();
+                        }
+                        ParsedCommandLineText(commandDescription.data());
+                    }
+                }
+                else
+                {
+                    ParsedCommandLineText(RS_(L"CommandPalette_FailedParsingCommandLine") + L"\n\t" + til::u8u16(_appArgs.GetExitMessage()));
+                }
+                _noMatchesText().Visibility(Visibility::Visible);
```

Comment:
Curious -- we're showing the no matches string all the time now? What's this look like?

Reply:
@DHowett - I am afraid this might be a leftover.. 🤦  which is hidden by the ParsedCommandLineText... 🤔 
Fixing it immediately, after I am finalizing the live search for initial review. Hopefully today (it is already 2am for me :smile:)

Reply:
Yep. It is a leftover. Thanks!